README.md

```diff
@@ -2,27 +2,52 @@
 
 This repository contains a sample app in Swift using the [Whereby iOS SDK](https://github.com/whereby/ios-sdk).
 
-Other platforms: 
+Other platforms:
+
 - [Android SDK](https://github.com/whereby/android-sdk)
 - [Browser SDK](https://github.com/whereby/browser-sdk)
 
 ## Prerequisites
+
 - The latest stable version of [Xcode](https://apps.apple.com/us/app/xcode/id497799835)
 - The latest stable version of [cocoapods](https://cocoapods.org/)
 - Sign up to [Whereby Embedded](https://whereby.com/information/embedded/) account
 - [Create a room](https://docs.whereby.com/creating-and-deleting-rooms) in your Whereby Embedded account
 
-## Running the app
+## Using the app with Cocoapods
+
 1. Install WherebySDK dependency using cocoapods. Starting at the root folder of this repo:
+
 ```
-cd WherebySDKDemo
+cd ~/ios-sdk-demo-main/Demo-CocoaPods
 pod install
 ```
+
 2. Open `WherebySDKDemo.xcworkspace` in Xcode.
 3. In `DemoViewController.swift` provide your room URL to open in the demo app.
 4. Run the project.
 
+## Using the app with Swift Package Manager
+
+1. Navigate to the Demo-SwiftPM folder:
+
+```
+cd ~/ios-sdk-demo-main/Demo-SwiftPM
+```
+
+2. Open `WherebySDKDemo.xcworkspace` in Xcode.
```

Comment:
I think there is a confusion here, the `WherebySDKDemo.xcworkspace` should be used when using cocoapods. 
When using SPM (Swift Package Manager), you should be able to use the `WherebySDKDemo.xcodeproj` file. 
If you just clone the repo, navigate to `Demo-SwiftPM`, open `WherebySDKDemo.xcodeproj` and press CMD-R, the app should run normally (and the package should be fetched automatically, no need to add it).